README.md

```diff
@@ -47,20 +47,20 @@ See the [API Documentation](api.md) for full details.
 The AIO IMS Library transparently maintains the login configuration and keep
 access and refresh tokens for reuse before they expire.
 
-All configuration is stored in a single `$ims` root property.
+All configuration is stored in a single `ims` root property.
 
 The library supports maintaining multiple configurations for different use cases.
-Each such configuration is stored in its own named object with the `$ims` configuration.
+Each such configuration is stored in its own named object with the `ims` configuration.
 Such a configuration is called an _IMS (configuration) context_ and has a label which allows to refer to the configuration by name.
 
 To simplify usage, there may be a designated _current context_ which is always used if explicit context is not given to the command.
-Inside the `$ims` configuration object, the name of the _current context_ is stored in the `$current` property.
+Inside the `ims` configuration object, the name of the _current context_ is stored in the `current` property.
 
-Here is an example `$ims` configuration
+Here is an example `ims` configuration
 
 ```js
 {
-  $ims: {
+  ims: {
```

Comment:
the new config looks like this https://github.com/adobe/aio-cli-plugin-auth/issues/43

Reply:
argh didn't notice the `config` subkey... will do the change, thanks for pointing this out

Reply:
@shazron what about the `cli` key does it goes into `contexts` or `config` ?

Reply:
contexts